raiden/tests/unit/test_pfs_integration.py

```diff
@@ -179,11 +179,19 @@ def get_best_routes_with_iou_request_mocked(
         amount,
         iou_json_data=None,
 ):
-    iou_response = Mock()
-    iou_response.configure_mock(status_code=200)
-    iou_response.json = Mock(return_value=iou_json_data or {})
 
-    with patch.object(requests, 'get', return_value=iou_response) as patched:
+    def iou_side_effect(*_, **kwargs):
+        body = kwargs.get('data', {})
+        required = ('sender', 'receiver', 'timestamp', 'signature')
+        message = 'Required argument in GET /iou call missing'
+        assert all(parameter in body for parameter in required), message
```

Comment:
If there's not other place where these args are tested, this would be a good place to add some basic checks (is_checksum_address, signature length) to avoid basic errors like not hex-encoded values.